api/chain.go

```diff
@@ -422,9 +421,16 @@ func (a *API) chainEstimateDateHandler(_ *apirest.APIdata, ctx *httprouter.HTTPC
 		return err
 	}
 	timestamp := a.vocinfo.HeightTime(height)
+	if timestamp == nil {
```

Comment:
FWIW I would keep HeightTime to returning a non-pointer, and you can use the IsZero method to check if it's valid/present or not.